src/templates/base/2019/base_chapter.html

```diff
@@ -61,13 +61,23 @@
 {# Calls to action for readers who want to engage more with this chapter. #}
 {% macro render_actions() %}
   <a class="alt btn" href="https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}">
-    <img src="/static/images/discuss.png" alt="{{ self.discuss_this_chapter() }}" height="24" width="24" loading="lazy" /><span id="num_comments"></span> <span data-translation id="comment-singular">{{ self.comment() }}</span><span data-translation id="comment-plural">{{ self.comments() }}</span>
+    <svg width="18" height="18">
+      <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#comment"></use>
+    </svg>    
+    <span id="num_comments"></span> <span data-translation id="comment-singular">{{ self.comment() }}</span>
+    <span data-translation id="comment-plural">{{ self.comments() }}</span>
   </a>
   <a class="alt btn" href="{{ metadata.get('results') }}/">
-    <img src="/static/images/results.png" alt="" height="24" width="24" loading="lazy" />{{ self.results() }}
+    <svg width="18" height="18">
```

Comment:
As this image is decorative we need to hide it since we are not giving a description.
```suggestion
    <svg width="18" height="18" aria-hidden="true">
```